example/public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
```

Comment:
is it necessary for local example?

Reply:
It came with CRA, I'll leave it there in case we want to deploy the example project